### PR TITLE
Changed license to Apache 2.0

### PR DESCRIPTION
**Pull request by coot**

### CODE REVIEW

File: nix/.stack.nix/cardano-node.nix [hunk 1]
```diff
@@ -4,8 +4,8 @@
     package = {
       specVersion = "1.10";
       identifier = { name = "cardano-node"; version = "3.0.1.87"; };
-      license = "MIT";
-      copyright = "2019 IOHK";
+      license = "NONE";
```
Comment (mrBliss) [inline nix/.stack.nix/cardano-node.nix]: Same here.

File: cardano-node.cabal [hunk 2]
```diff
@@ -4,8 +4,10 @@ description:    The cardano full node
 author:         IOHK
 maintainer:     operations@iohk.io
 copyright:      2019 IOHK
-license:        MIT
```
Comment (mrBliss) [inline cardano-node.cabal]: Missing:
```
license:             Apache-2.0
```

File: nix/.stack.nix/cardano-node.nix (hunk 1 above)
Comment (coot) [reply nix/.stack.nix/cardano-node.nix]: That's wired, let me check.

Comment (coot) [reply nix/.stack.nix/cardano-node.nix]: ok, I missed the `license` field in `cardano-node.cabal` file.

File: cardano-node.cabal [hunk 3]
```diff
@@ -4,8 +4,11 @@ description:    The cardano full node
 author:         IOHK
 maintainer:     operations@iohk.io
 copyright:      2019 IOHK
```
Comment (mrBliss) [inline cardano-node.cabal]: I just got this warning:
```
Cabal file warning in/home/thomas/work/cardano-node/cardano-node.cabal@6:1: The field "copyright" is specified more than once at positions 6:1, 11:1
```

**Pull request merged**